backend/src/v5/middleware/permissions/components/teamspaces.js

```diff
@@ -53,4 +53,15 @@ TeamspacePerms.isTeamspaceMember = async (req, res, next) => {
 	}
 };
 
+TeamspacePerms.hasMemberAccessToTeamspace = async (req, res, next) => {
```

Comment:
This middleware doesn't make much sense. If anything this should live in `dataConverter/input`

All the permissions middleware are about checking if the user has sufficient access rights for the action. This is checking if the data supplied is valid

Reply:
So either move this to dataConverter/input and call it something like `memberExists`

or just check the member exists before fetching the avatar in the processor